maps-compose-widgets/build.gradle

```diff
@@ -0,0 +1,51 @@
+plugins {
+    id 'kotlin-android'
+    id 'kotlin-parcelize'
+}
+
+android {
+    compileSdk 31
+
+    defaultConfig {
+        minSdk 21
+        targetSdk 31
+        versionCode 1
+        versionName "1.0"
+    }
+
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+
+    composeOptions {
+        kotlinCompilerExtensionVersion "$compose_version"
+    }
+
+    buildFeatures {
+        buildConfig false
+        compose true
+    }
+
+    kotlinOptions {
+        jvmTarget = '1.8'
+        freeCompilerArgs += '-Xexplicit-api=strict'
+        freeCompilerArgs += '-Xopt-in=kotlin.RequiresOptIn'
+    }
+}
+
+dependencies {
+    implementation "androidx.compose.foundation:foundation:$compose_version"
+    implementation 'com.google.maps.android:maps-compose:2.2.1'
```

Comment:
This should reference the local module so the artifact is always tied to the same version as `maps-compose`.

```
implementation project(':maps-compose')
```

Reply:
Done in https://github.com/googlemaps/android-maps-compose/pull/157/commits/33990bb781f9219e801eb5deb4bfe0c7207e4d62